cogent/Makefile

```diff
@@ -50,33 +41,49 @@ build: setup-config
 	$(E) "Building..."
 	$(CABAL) build
 
-configure: install-deps
+configure:
 	$(E) "Configuring"
 	$(CABAL) configure
-	$(CABAL) build
 
 sandbox:
-	$(E) "Building in a sandbox"
+ifneq ($(wildcard .cabal-sandbox/),)
```

Comment:
It doesn't seem to. And that's the reason, I added that. Especially when it adds `isa-parser`.